.github/workflows/test.yml

```diff
@@ -11,8 +11,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        # switch to `8` when https://github.com/actions/setup-node/issues/27 is fixed
-        node: [8.16.2, 10, 12, 14]
+        node: [12.20, 14]
```

Comment:
We can use `12` here as it'll use the latest version, which is greater than 12.20.